app/controllers/api/v1/moves_controller.rb

```diff
@@ -73,6 +74,25 @@ def find_move
           .includes(:from_location, :to_location, person: { profiles: %i[gender ethnicity] })
           .find(params[:id])
       end
+
+      def import_moves_from_nomis
+        return unless from_location_nomis_agency_id && date
```

Comment:
Surely effective, could be a good idea to have this in a private method called in a `before_action`.